backends/calico/client.go

```diff
@@ -162,7 +163,11 @@ func (c *client) OnStatusUpdated(status api.SyncStatus) {
 		defer c.cacheLock.Unlock()
 		c.synced = true
 		c.waitForSync.Done()
-		log.Debug("Data is now syncd, can start rendering templates")
+		log.Debugf("Data is now syncd, can start rendering templates")
+
+		// Now that we're in-sync, check if we should update our log level
```

Comment:
@robbrockbank we were missing this before - this lets us set log level when we connect to the datastore for the first time.

Reply:
D'oh yes!